build_tools/travis/install.sh

```diff
@@ -53,7 +53,7 @@ pip install -r requirements.txt
  # Install the following only if running tests
 if [[ "$SKIP_TESTS" != "true" ]]; then
     # PyTorch
-    conda install --yes pytorch -c pytorch
+    conda install --yes pytorch-nightly-cpu -c pytorch
```

Comment:
Make sure you communicate that this is what we expect users to install, if we're testing on it. In the future, we'll need to standardize on minor releases of PyTorch and use the import checks to guard against usage of potentially unsupported features.